source/transactions/tests/legacy/retryable-commit-forbid_serverless.yml

```diff
@@ -0,0 +1,410 @@
+runOn:
+    -
+        minServerVersion: "4.0"
+        topology: ["replicaset"]
+    -
+        minServerVersion: "4.1.8"
+        topology: ["sharded", "load-balanced"]
+        # This file contains tests originally found in retryable-commit.yml that
+        # are incompatible with serverless:
+        #
+        #  * Failing commitTransaction with closeConnection:true may abort the
+        #    transaction (CLOUDP-202309).
+        #  * Fail points using state change errors (e.g. InterruptedAtShutdown,
+        #    ShutdownInProgress) may remain enabled indefinitely (CLOUDP-84573).
```

Comment:
This comment addresses the discussion in https://github.com/mongodb/specifications/pull/1459#discussion_r1316000236